src/components/DictionaryDetail/DictionaryDetailPresentation.js

```diff
@@ -267,6 +269,22 @@ function DictionaryDetailPresentation({
                     </li>
                   ))
                 : null}
+              {entry?.relatedVideoLinks
+                ? entry?.relatedVideoLinks?.map((video) => (
+                    <li key={video.id} className="my-2 p-3">
+                      <div className="rounded-lg relative pb-[56.25%] h-0">
```

Comment:
since this `pb-[56.25%]` value is being used multiple times, it can be added to the `tailwind.config.js` and then used across the project.
Similar for any other custom values which are being used in more than one place.